sdk/core/core/inc/az_http_transport.h

```diff
@@ -85,12 +85,19 @@ AZ_NODISCARD az_result
 az_http_request_get_header(_az_http_request const* request, int32_t index, az_pair* out_header);
 
 /**
- * @brief Get parts of an HTTP request.
+ * @brief Get parts of an HTTP request. `NULL` in accepted to ignore getting any parts, for example,
```

Comment:
> `NULL` in accepted to ignore getting any parts,

Should the `in` be `is`?

```suggestion
 * @brief Get parts of an HTTP request. `NULL` is accepted to ignore getting any parts, for example,
```